mne/viz/tests/test_topo.py

```diff
@@ -77,6 +77,11 @@ def test_plot_topo():
     # Show topography
     evoked = _get_epochs().average()
     plot_evoked_topo(evoked)  # should auto-find layout
+    # Test jointplot
+    evoked.plot_joint()
+    evoked.plot_joint(title='test', ts_args=dict(spatial_colors=True),
+                      topomap_args=dict(colorbar=True, times=[0.]))
```

Comment:
I've added a new example. It bothered me that plot_evoked_erf_erp returned 4 figures, and I also thought the dict arguments needed a dedicated example.